Library/Homebrew/os/linux/elf.rb

```diff
@@ -172,6 +193,12 @@ def needed_libraries_using_readelf(path)
     end
   end
 
+  def patcher
+    raise NotImplementedError unless defined?(PatchELF)
```

Comment:
Do we need this `raise` + `defined?` check? `PatchELF` should always be defined if the surrounding invariant (`HOMEBREW_READELF_RB`) holds; an error here would indicate a developer mistake.

Reply:
yes, this was a sort of defensive programming, to stop myself from using it in places, when patchelf is not defined. will remove it in next push